ij-server/src/main/kotlin/org/jetbrains/research/ij/headless/server/SingleFileProjectConfigurator.kt

```diff
@@ -0,0 +1,43 @@
+package org.jetbrains.research.ij.headless.server
+
+import com.intellij.lang.Language
+import com.intellij.openapi.Disposable
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFile
+import org.jetbrains.research.ij.headless.server.utils.findPsiFileByName
+import org.slf4j.LoggerFactory
+import java.util.concurrent.atomic.AtomicReference
+
+object SingleFileProjectConfigurator {
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    data class SingleFileProject(
+        val language: Language,
+        val project: Project,
+        val file: PsiFile,
+        private val disposable: Disposable
+    )
+
+    fun configureProject(project: Project, language: Language, disposable: Disposable): SingleFileProject {
+        logger.info("Start to create a new file for language \"${language.id}\"")
+
+        val file = AtomicReference<PsiFile>()
+
+        ApplicationManager.getApplication().invokeAndWait {
+            val mainPsiFile = when (language.id) {
+                LanguageId.Python.name -> findPsiFileByName(project, "main.py")
```

Comment:
I'd prefer to move file names into consts and refer to these consts in the readme